### ISSUE TITLE
Output a ResponseLoadSet from fit_yield_functions task

### ISSUE THREAD

**Issue by aplowman** [LightForm-group/matflow-formable#1]
This will make it easier to visualise the yield stress states together with the fitted yield functions.

**Comment by ElliotCN** [comment]
I have changed the output to a dictionary, not sure if there is anything else you think I should add @aplowman  ? One thing I was wondering was how to output show_yield_functions_2D & show_yield_functions_3D which are properties of the load response set.

**Comment by aplowman** [comment]
Sorry, I didn't stay on top of this. Thanks for your efforts @ElliotCN.

This works in the recent releases of formable/matflow-formable. Given a loaded workflow, `workflow`, we can get at the `LoadResponseSet` like this:

```py
from matflow import load_workflow
from formable.load_response import LoadResponseSet

response_set_dict = workflow.tasks.fit_yield_function.elements[0].outputs.fitted_yield_functions
response_set = LoadResponseSet.from_dict(response_set_dict)
```

We can plot the individual yield functions like this:

```py
my_yield_func = response_set.fitted_yield_functions[0]['yield_function']
my_yield_func.show_2D([0, 0, 1])
```

Or if you want to include information from the `LoadResponseSet` (like the stress states used to fit), we can do:

```py
response_set.show_yield_functions_2D([0, 0, 1], show_stress_states=True)
```

Fixed in [v0.1.8](https://github.com/LightForm-group/matflow-formable/tree/v0.1.8)
